src/apps/lwaftr/rangemap.lua

```diff
@@ -83,7 +83,7 @@ function RangeMapBuilder:add(key, value)
    self:add_range(key, key, value)
 end
 
-function RangeMapBuilder:build(default_value)
+function RangeMapBuilder:build(default_value, ignore_duplicates)
```

Comment:
I am pretty sure that we don't want to ignore duplicates -- rather we want to require that all entries with the same key have the same value.  Maybe that means the builder should use a ctable as its intermediate data store and verify that when a key is added but already present in the builder, that the value is the same.